Dockerfile

```diff
@@ -83,23 +87,29 @@ RUN pip3 install \
     hidapi \
     noiseprotocol \
     protobuf \
+    mypy-protobuf \
```

Comment:
why move this here? imho the pip version below is better because it pins the version, and it's easier to maintain because all the python packages are in one place.

Reply:
my next pr will use `python3 pip install -r requirements.txt` so that we don't have that list in two places.

Reply:
did it in this PR.